src/devices/Ahtxx/Aht20.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.I2c;
+
+namespace Iot.Device.Ahtxx
+{
+    /// <summary>
+    /// AHT20 temperature and humidity sensor binding.
+    /// </summary>
+    public class Aht20 : AhtBase
```

Comment:
I'd prefer base class was called Aht20 but if you prefer this way at least make the base abstract

Reply:
Done